documentation/course/e2/README.md

```diff
@@ -2159,7 +2164,53 @@ TODO @piterek130: Dodać diagram pakietów, opis architektury i endpointy.
+| `admin`                                     | `DELETE`   | `/ext/v1/stops/:id`     | `LOG/10`                               | Deaktywacja przystanku.         |
+| `passenger`, `admin`                        | `GET`      | `/ext/v1/routes`¹       | `LOG/11`, `LOG/12`                     | Pobranie listy kursów.          |
+| `passenger`, `admin`                        | `GET`      | `/ext/v1/routes/:id`    | `LOG/11`, `LOG/12`                     | Pobranie informacji o kursie.   |
+| `admin`                                     | `POST`     | `/ext/v1/routes`        | `LOG/13`,                              | Utworzenie nowego kursu.        |
+| `admin`                                     | `PATCH`    | `/ext/v1/routes/:id`    | `LOG/14`                               | Edytowanie kursu.               |
+| `admin`                                     | `DELETE`   | `/ext/v1/routes/:id`    | `LOG/15`                               | Deaktywacja kursu.              |
+| `admin`                                     | `GET`      | `/ext/v1/vehicles`¹     | `LOG/16`                               | Pobranie listy pojazdów.        |
+| `admin`                                     | `POST`     | `/ext/v1/vehicles`      | `LOG/17`                               | Utworzenie nowego pojazdu.      |
+| `admin`                                     | `PATCH`    | `/ext/v1/vehicles/:id`  | `LOG/18`                               | Edytowanie pojazdu.             |
+| `admin`                                     | `DELETE`   | `/ext/v1/vehicles/:id`  | `LOG/19`                               | Deaktywacja pojazdu.            |
+| `passenger`, `driver`, `inspector`, `admin` | `GET`      | `/ext/v1/accidents`¹    | `LOG/20`, `LOG/21`, `LOG/24` `LOG/27`, | Pobranie listy awarii.          |
+| `passenger`, `driver`, `inspector`, `admin` | `GET`      | `/ext/v1/accidents/:id` | `LOG/20`, `LOG/21`, `LOG/24` `LOG/27`, | Pobranie informacji o awarii.   |
+| `driver`, `inspector`                       | `POST`     | `/ext/v1/accidents`     | `LOG/22`, `LOG/25`                     | Zgłoszenie nowej awarii.        |
+| `driver`, `inspector`                       | `PATCH`    | `/ext/v1/accidents/:id` | `LOG/23`, `LOG/26`                     | Edytowanie awarii.              |
+
+¹ - endpoint wspiera paginację oraz filtrowanie.
+
+#### API wewnętrzne
+
+| **Metoda** | **Endpoint**                 | **Producent** | **Konsument** | **Opis**                                                                       |
+| ---------- | ---------------------------- | ------------- | ------------- | ------------------------------------------------------------------------------ |
+| `GET`      | `/int/v1/health`             | Leprechaun    | —             | Sprawdzenie stanu głównego serwisu ([`M/03`](#m03-healthchecki-dla-serwisów)). |
```

Comment:
Please add `/int/v1/endpoints` similar to the one on my branch, we discussed it yesterday.